tests/ai_safety/evalhub/mcp/test_evalhub_mcp_route.py

```
@@ -82,8 +83,16 @@ def test_evalhub_mcp_configmap_exists(
-        data = configmap.instance.data or {}
-        assert "config.yaml" in data, f"Expected config.yaml in {configmap_name}"
+        try:
+            for data in TimeoutSampler(
+                wait_timeout=60,
+                sleep=5,
+                func=lambda: configmap.instance.data or {},
+            ):
+                if "config.yaml" in data:
+                    break
+        except TimeoutExpiredError:
+            raise AssertionError(f"Expected config.yaml in {configmap_name}")
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)